modules/specialized-guides/pages/large-deployments/hub-install.adoc

```diff
@@ -1,35 +1,75 @@
+. Preliminary Requirement: CA Certificate.
+. On every peripheral server, copy the same CA to [path]``/etc/pki/trust/anchors/`` and run ``update-ca-certificates``.
+. You can now install {productname} using the following command (replace appropriately the names of the certificates):
++
+
+----
+mgradm install podman --ssl-ca-root CA-Certificate.crt --ssl-server-cert server.crt --ssl-server-key server.key
+----
+
+
+.Procedure: Using Certificates Self-Generated with [command]``rhn-ssl-tool``
+. Preliminary Requirement: Hub server installation
+
+. On the hub server, enter the hub container with:
++
 
+----
+mgrctl term
+----
+
+
+. Run [command]``rhn-ssl-tool`` for every pheripheral server:
++
+
+----
+rhn-ssl-tool --gen-server --dir="/root/ssl-build" --set-country="COUNTRY" \
+  --set-state="STATE" --set-city="CITY" --set-org="ORGANIZATION" \
+  --set-org-unit="ORGANIZATION UNIT" --set-email="name@example.com" \
+  --set-hostname=<hostname> --set-cname="example.com"
+----
 
-.Procedure: Installing and Configuring the Hub XMLRPC API
-. On the Hub Server, or on a host that has access to all peripheral servers' XMLRPC APIs, install the ``hub-xmlrpc-api`` package.
-    The package is available in the {productname} {productnumber} repositories.
-. OPTIONAL: Set the Hub XMLRPC API service to start automatically at boot time, and start it immediately:
+. On every peripheral server:
+* Copy [path]``/root/ssl-build/RHN-ORG-TRUSTED-SSL-CERT`` and  [path]``/root/ssl-build/<hostname>/server.crt`` and [path]``/root/ssl-build/<hostname>/server.key`` to the peripheral server host.
+* On every peripheral server, copy [path]``RHN-ORG-TRUSTED-SSL-CERT`` to [path]``/etc/pki/trust/anchors/``, and run [command]``update-ca-certificates``.
+* Install {productname} with:
 +
+
+----
+mgradm install podman --ssl-ca-root RHN-ORG-TRUSTED-SSL-CERT --ssl-server-cert server.crt --ssl-server-key server.key
+----
+
+* Register the peripheral server to the hub server:
++
+
 ----
-sudo systemctl enable hub-xmlrpc-api.service
-sudo systemctl start hub-xmlrpc-api.service
+mgradm hub register --api-password <hub password> --api-server <hub fqdn> --api-user <hub admin>
 ----
 
-. OPTIONAL: Check that these parameters in the [path]``/etc/hub/hub.conf`` configuration file are correct:
-*  ``HUB_API_URL``: URL to the Hub Server XMLRPC API endpoint.
-    Use the default value if you are installing `hub-xmlrpc-api` on the Hub Server.
-* ``HUB_CONNECT_TIMEOUT``: the maximum number of seconds to wait for a response when connecting to a Server.
-    Use the default value in most cases.
-* ``HUB_REQUEST_TIMEOUT``: the maximum number of seconds to wait for a response when calling a Server method.
-    Use the default value in most cases.
-* ``HUB_CONNECT_USING_SSL``: use HTTPS instead of HTTP for communicating with peripheral Servers.
-    Recommended for a secure environment.
```

Comment:
Do you want to drop this list of options completely?
In this case the section below about `/etc/systemd/system/uyuni-hub-xmlrpc.service.d/local.conf` is not needed.

Reply:
no, it should stay.  thanks for spotting and reporting!